uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -0,0 +1,251 @@
+define(function () {
+  return {
+    all: function (params, withBalance) {
```

Comment:
disagree - the mocked data can change its behavior but that should not affect the service layer and hence redirected to a method